tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/ForwardGet.scala

```diff
@@ -0,0 +1,44 @@
+package com.linagora.tmail.james.jmap.model
+
+import eu.timepit.refined.auto._
+import org.apache.james.core.MailAddress
+import org.apache.james.jmap.core.Id.Id
+import org.apache.james.jmap.core.{AccountId, Id, Properties, UuidState}
+import org.apache.james.jmap.method.WithAccountId
+
+case class UnparsedForwardId(id: Id)
+
+case class ForwardIds(value: List[UnparsedForwardId])
+
+case class ForwardGetRequest(accountId: AccountId,
+                             ids: Option[ForwardIds],
+                             properties: Option[Properties]) extends WithAccountId
+
+case class ForwardNotFound(value: Set[UnparsedForwardId]) {
+  def merge(other: ForwardNotFound): ForwardNotFound = ForwardNotFound(this.value ++ other.value)
+}
+
+case class ForwardId()
```

Comment:
case object?

Reply:
Hmm rationals behind that? I mostly sticked to something similar to VacationResponseGetMethod as it's a simple singleton too. 

`ForwardId` seems used like for the serialization for example. Is case object justified? (maybe it is, I'm just not sure :))

Reply:
> . Is case object justified? (maybe it is, I'm just not sure :))

Yes that will empower us to not create tons of ForwardId instances...